utils/config.go

```diff
@@ -794,6 +799,16 @@ func (cc *configContext) setChainId() error {
 	return nil
 }
 
+// openBaseDB opens existing database. If it does not exist error is returned instead.
+func openBaseDB(path string) (db.BaseDB, error) {
+	_, err := os.Stat(path)
+	if err != nil {
+		return nil, err
+	}
+
+	return db.NewDefaultSubstateDB(path)
```

Comment:
This function does an additional path check. NewDefefaultSubstateDB should already return an error if path does exist. Why is the additional check needed?